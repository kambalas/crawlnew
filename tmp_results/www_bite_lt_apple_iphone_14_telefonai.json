[
  {
    "name": "iPhone 14",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 14 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 14 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 14 Pro Max",
    "price": "",
    "error": false
  },
  {
    "index": 0,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  }
]