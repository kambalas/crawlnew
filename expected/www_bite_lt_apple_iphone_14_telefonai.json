[
  {
    "name": "Apple iPhone 14 telefonai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 128 GB talpos telefonai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 256 GB talpos telefonai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus 512 GB talpos telefonai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus 256 GB talpos telefonai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus 128 GB talpos telefonai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Red / raudoni telefonai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Purple / purpuriniai telefonai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Blue / mėlyni telefonai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Midnight / juodi telefonai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Yellow / geltoni telefonai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Starlight / balti telefonai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus Red / raudoni telefonai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus Purple / purpuriniai telefonai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus Blue / mėlyni telefonai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus Yellow / geltoni telefonai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro telefonai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max telefonai",
    "price": "",
    "error": false
  },
  {
    "index": 2,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  }
]